.github/workflows/cicd.yml

```
@@ -0,0 +1,81 @@
+  pull-requests: write
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 30: did not find expected key

(syntax-check)